src/Cli/Microsoft.DotNet.Cli.Utils/MSBuildForwardingAppWithoutLogging.cs

```diff
@@ -17,6 +17,7 @@ internal class MSBuildForwardingAppWithoutLogging
     {
         private static readonly bool AlwaysExecuteMSBuildOutOfProc = Env.GetEnvironmentVariableAsBool("DOTNET_CLI_RUN_MSBUILD_OUTOFPROC");
         private static readonly bool UseMSBuildServer = Env.GetEnvironmentVariableAsBool("DOTNET_CLI_USE_MSBUILD_SERVER", false);
+        private static readonly string TerminalLoggerDefault = Env.GetEnvironmentVariable("DOTNET_CLI_BUILD_TERMINAL_LOGGER");
```

Comment:
I see, but `DOTNET_CLI_USE_MSBUILD_TERMINAL_LOGGER` implies boolean, like use or dontuse, but this variable has null | true | false | on | off | auto states. 
Would `DOTNET_CLI_CONFIGURE_MSBUILD_TERMINAL_LOGGER` be better fitting? Not that it matter as mach as most people whom will need it will simple copy paste from docs page.

Reply:
`DOTNET_CLI_CONFIGURE_MSBUILD_TERMINAL_LOGGER` is acceptable to me

Reply:
done